slsim/image_simulation.py

```diff
@@ -266,35 +279,41 @@ def point_source_image_without_variability(
     :param mag_zero_point: magnitude zero point in band
     :param delta_pix: pixel scale of image generated
     :param num_pix: number of pixels per axis
-    :param psf_kernels: psf kernels.
+    :param psf_kernel: psf kernel for an image.
+    :param transform_pix2angle: transformation matrix (2x2) of pixels into coordinate
+        displacements
     :return: point source images
     """
-
-    image_data = point_source_coordinate_properties(
-        lens_class=lens_class,
-        band=band,
-        mag_zero_point=mag_zero_point,
-        delta_pix=delta_pix,
-        num_pix=num_pix,
-        transform_pix2angle=transform_pix2angle,
-    )
+    kwargs_model, kwargs_params = lens_class.lenstronomy_kwargs(band="i")
+    kwargs_ps = kwargs_params["kwargs_ps"]
 
     data_class = image_data_class(
         lens_class, band, mag_zero_point, delta_pix, num_pix, transform_pix2angle
     )
 
-    ra_image_values = image_data["ra_image"]
-    dec_image_values = image_data["dec_image"]
-    psf_class = []
-    for i in range(len(psf_kernels)):
-        psf_class.append(PSF(psf_type="PIXEL", kernel_point_source=psf_kernels[i]))
+    psf_class = PSF(psf_type="PIXEL", kernel_point_source=psf_kernel)
+    if kwargs_ps is not None:
+        image_data = point_source_coordinate_properties(
+            lens_class=lens_class,
+            band=band,
+            mag_zero_point=mag_zero_point,
+            delta_pix=delta_pix,
+            num_pix=num_pix,
+            transform_pix2angle=transform_pix2angle,
+        )
+        ra_image_values = image_data["ra_image"]
+        dec_image_values = image_data["dec_image"]
+        magnitude = lens_class.point_source_magnitude(band, lensed=True)
+        amp = magnitude_to_amplitude(magnitude, mag_zero_point)
+    else:
+        ra_image_values = np.zeros(1)
```

Comment:
is this desired? Perhaps you can have empty lists instead?

Reply:
We just want to produce a empty image. I changed it little bit.